src/components/SignUp/SignUpForm.js

```diff
@@ -0,0 +1,132 @@
+import React, { useState, useCallback } from 'react';
+import Airtable from 'airtable';
+import TextField from '@material-ui/core/TextField';
+import MenuItem from '@material-ui/core/MenuItem';
+import InputLabel from '@material-ui/core/InputLabel';
+import Select from '@material-ui/core/Select';
+import './SignUp.css';
+import Grid from '@material-ui/core/Grid';
+import Button from '@material-ui/core/Button';
+
+const airtableConfig = {
+  apiKey: process.env.REACT_APP_AIRTABLE_USER_KEY,
+  baseKey: process.env.REACT_APP_AIRTABLE_BASE_KEY,
+};
+
+const base = new Airtable({ apiKey: airtableConfig.apiKey }).base(airtableConfig.baseKey);
+
+const INITIAL_FORM_STATE = {
+  username: '',
+  fullname: '',
+  password: '',
+};
+
+// Main fields are, username (email), password, First Name, Last Name, Role
```

Comment:
For sake of standardization, make sure to remove comments unless there is a really ambiguous function or set of lines that require explicit descriptions